firebase.libsonnet

```diff
@@ -0,0 +1,51 @@
+// jsonnet config for firebase configuration
+//
+// $ make config
+
+// Some functions to generate redirect configs
+local redirect = {
+    files_template: "https://raw.githubusercontent.com/foxygoat/%s/:path",
+
+    // temporary generates a redirect config for a temporary (302) redirect
+    // of s -> d.
+    temporary(s, d):: {
+        source: s,
+        destination: d,
+        type: 302,
+    },
+
+    // files generates a redirect for serving files from a github repo.
+    // For example, you can import a jsonnet file using
+    // `foxygo.at/jsonnext/importer` using
+    // `//foxygo.at/jsonnext/master/jlib/utils.libsonnet` as a filename. This
+    // gets redirected to a raw github file in the repo under the `master` ref.
+    files(repo):: self.temporary("%s/:path*" % repo, self.files_template % repo),
+};
+
+local html = {
+    index(module_name): |||
+        <html><head>
```

Comment:
Could you add an html comment: generated by jsonnet or something, please?

Reply:
done